### PR TITLE
Updated licenses

### PR DESCRIPTION
**Pull request by Wild8ill**

### CODE REVIEW

**Review by hakonfam — APPROVED** [review]
Files which I am codeowner for looks OK.

File: doc/mcuboot/conf.py [hunk 1]
```diff
@@ -1,3 +1,9 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
 #!/usr/bin/env python3
```
Comment (KAGA164) [inline doc/mcuboot/conf.py:7]: I wonder if these scripts will work if the Shebang line `#!/usr/bin/env python3` isn't the first line of the file and python script has a C style comments ?

**Review by KAGA164 — CHANGES_REQUESTED** [review]

File: samples/openthread/cli/harness/nrf52840-ncs.py [hunk 2]
```diff
@@ -1,3 +1,9 @@
+/*
```
Comment (MarekPorwisz) [inline samples/openthread/cli/harness/nrf52840-ncs.py:1]: I see three issues here:
1. I think #!/usr/bin/env python must be the first line in the file, the copyright may come after
2. The file already has the copyrights done the proper way. This is not a nordic file in fact, so I am not sure this should be changed
3. The copyrights should be in python style comments not C

**Review by MarekPorwisz — CHANGES_REQUESTED** [review]
I've logged an issue to the files I am an owner.

In general I think that python files shall use Python style comments, not the C style.

**Review by sigvartmh — APPROVED** [review]
Changes to files I've authored seems good

**Review by MarekPorwisz — APPROVED** [review]

**Comment by carlescufi** [comment]
@Wild8ill can we close this one and send Pull Requests for the different components? A couple of them have been merged already.

**Comment by shantha-14** [comment]
PR closed without merging. Domain owners are expected to raise individual PRS

**Pull request closed without merging**